http/cookie_test.ts

```diff
@@ -177,6 +177,15 @@ Deno.test({
       "Space=Cat; Secure; HttpOnly; Max-Age=2",
     );
 
+    res.headers = new Headers();
+    setCookie(res, {
+      name: "Space",
+      value: "Cat",
+      httpOnly: true,
+      secure: true,
+      maxAge: 0,
+    });
```

Comment:
There should probably be an assertion here, either check for the value of the header or that it doesn't throw